docs/cli.rst

```diff
@@ -24,17 +24,32 @@ and providing a simple YAML configuration file:
 .. code-block:: yaml
 
     # foobar.yaml
-
     AMQP_URI: 'amqp://guest:guest@localhost'
-
     WEB_SERVER_ADDRESS: '0.0.0.0:8000'
-
     rpc_exchange: 'nameko-rpc'
-
     max_workers: 10
-
     parent_calls_tracked: 10
 
+    LOGGING:
```

Comment:
could we get away with a simpler example here? this increases what we describe as a "simple yaml config file" from 5 to 20 lines